osquery/tables/system/darwin/keychain_items.cpp

```diff
@@ -55,9 +55,17 @@ void genKeychainItem(const SecKeychainItemRef& item, QueryData& results) {
   // Any tag that does not exist for the item will prevent the entire result.
   for (const auto& attr_tag : kKeychainItemAttrs) {
     tags[0] = attr_tag.first;
-    SecKeychainItemCopyAttributesAndData(
+    auto status = SecKeychainItemCopyAttributesAndData(
```

Comment:
I suggest that we use `os_status` as the variable name here. This way we are more in line with what the rest of codebase that use Security/keychain APIs. This way we also don't confuse it with osquery's own `Status` class.

Reply:
Thanks, I made this change